sessions/js-array-methods-2/zoo-animals/js/index.js

```diff
@@ -0,0 +1,37 @@
+import { zooAnimals } from "../utils/db.js";
+
+// This website uses a function to check wether a specific animal is part of the zoo or not but it's not working correct.
+// Please implement the function 'checkAnimal' that gets an array of animals and an animal name
+// The function should return 'true' if the animal name is included in the array or 'false' if not.
+
+function checkAnimal(animals, animalName) {
```

Comment:
I decided to use `animals` as a parameter because first this makes the function pure and second: I love your idea of making it case insensitive and made a bonus challenge for that. This means students can modify the array temporary inside the function.